src/components/MenuBar/MenuBar.test.tsx

```diff
@@ -291,4 +299,158 @@ describe('MenuBar', () => {
+      activity_type: 'Quiz',
+      classification: 'Formative',
+      created_at: '2023-04-19T10:30:00.000Z',
+      created_by: 'John Doe',
+      id: 123,
+      last_updated: '2023-04-20T15:45:00.000Z',
+      lms_id: 456,
+      name: 'Quiz on Chapter 3',
+      student_learning_element: null,
+      university: 'ABC University'
+    }
+
+    const exampleLearningElement2: LearningElement = {
+      activity_type: 'Quiz',
+      classification: 'Formative',
+      created_at: '2023-04-19T10:30:00.000Z',
+      created_by: 'John Doe',
+      id: 123,
+      last_updated: '2023-04-20T15:45:00.000Z',
+      lms_id: 456,
+      name: 'Quiz on Chapter 5',
+      student_learning_element: null,
+      university: 'ABC University'
+    }
+
+    const topics: Topic[] = [
+      {
+        contains_le: true,
+        created_at: '2021-09-01T12:00:00.000Z',
+        created_by: 'dimitri',
+        id: 1,
+        is_topic: true,
+        last_updated: '2021-09-01T12:00:00.000Z',
+        lms_id: 1,
+        name: 'Allgemeine Informatik',
+        parent_id: 1,
+        student_topic: {
+          done: false,
+          done_at: null,
+          id: 1,
+          student_id: 1,
+          topic_id: 1,
+          visits: []
+        },
+        university: 'HS-KE'
+      },
+      {
+        contains_le: true,
+        created_at: '2021-09-01T12:00:00.000Z',
+        created_by: 'dimitri',
+        id: 2,
+        is_topic: true,
+        last_updated: '2021-09-01T12:00:00.000Z',
+        lms_id: 1,
+        name: 'Zustand',
+        parent_id: 1,
+        student_topic: {
+          done: false,
+          done_at: null,
+          id: 1,
+          student_id: 1,
+          topic_id: 1,
+          visits: []
+        },
+        university: 'HS-KE'
+      }
+    ]
+
+    const learningElementPath: LearningPath[] = [
+      {
+        based_on: 'some-Algorithm',
+        calculated_on: 'today',
+        course_id: 1,
+        id: 1,
+        path: [
+          {
+            id: 1,
+            learning_element: exampleLearningElement1,
+            learning_element_id: 1,
+            learning_path_id: 1,
+            position: 1,
+            recommended: true
+          }
+        ]
+      },
+      {
+        based_on: 'some-Algorithm',
+        calculated_on: 'today',
+        course_id: 1,
+        id: 2,
+        path: [
+          {
+            id: 2,
+            learning_element: exampleLearningElement2,
+            learning_element_id: 1,
+            learning_path_id: 1,
+            position: 1,
+            recommended: true
+          }
+        ]
+      }
+    ]
+
+    const mockUseLearningPath = jest.fn().mockReturnValue({
+      loading: false,
+      topics: topics,
+      learningPaths: learningElementPath
+    })
+
+    const props: MenuBarProps = {
+      useLearningPathTopic: mockUseLearningPath
+    }
+
+    const { getAllByText, getByText } = render(
+      <Router location={history.location} navigator={history}>
```

Comment:
Hier memory router nutzen. Router sollte man nicht in tests nutzen^^
```suggestion
      <MemoryRouter location={history.location} navigator={history}>
```

Reply:
<MemoryRouter initialEntries={['/home']}>

damit funktioniert der Test nicht so wie zuvor, bzw er navigiert nicht mehr und bleibt bei /home nachm fireEvent.click

Reply:
Wie greife ich beim MemoryRouter auf den Pfad zu, nachdem er geändert wurde 🤔

Reply:
Musst nachschauen. Kp leider haha

Reply:
https://github.com/search?q=repo%3Aremix-run%2Freact-router%20MemoryRouter&type=code

Habe deren Tests durchgeschaut, die scheinen nie auf den Pfad zuzugreifen. Sondern nur auf den Text oder sonstiges was auf der neuen Page enthalten sein soll. 
Ich finde es eher ein Downgrade zu davor

Reply:
Wurde mit useNavigate und SpyOn / MemoryRouter gelöst 👍